src/prevector.h

```diff
@@ -397,14 +397,16 @@ class prevector {
     inline void resize_uninitialized(size_type new_size) {
         // resize_uninitialized change the size of the prevector but does not initialize.
         // If size < new_size, the added elements must be initialized explicitly after it returns.
-        difference_type count = new_size - size();
+        size_type cur_size = size();
         if (capacity() < new_size) {
             change_capacity(new_size);
+            _size += new_size - cur_size;
+            return;
         }
-        if (count < 0) {
+        if (new_size < cur_size) {
             erase(item_ptr(new_size), end());
         } else {
-            _size += count;
+            _size += new_size - cur_size;
```

Comment:
@kallewoof Now I mesured a bench that using size() instead of cur_size;
I am surprised that compiler optimization is excellent.
In this patch, rather than referring to local variables, function calls are faster.